<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgUserEdit</class>
 <widget class="QDialog" name="dlgUserEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit User</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblUserId">
       <property name="text">
        <string>User Id:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lblUserIdValue">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblName">
       <property name="text">
        <string>Login &amp;Name:</string>
       </property>
       <property name="buddy">
        <cstring>ledName</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ledName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblPwd">
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="ledPwd">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblRePwd">
       <property name="text">
        <string>Retype Password:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="ledRePwd">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblRealName">
       <property name="text">
        <string>&amp;Real Name:</string>
       </property>
       <property name="buddy">
        <cstring>ledRealName</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="ledRealName"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lblGroup">
       <property name="text">
        <string>User &amp;Group:</string>
       </property>
       <property name="buddy">
        <cstring>cmbGroup</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="cmbGroup"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lblActive">
       <property name="text">
        <string>&amp;Active:</string>
       </property>
       <property name="buddy">
        <cstring>chbActive</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="chbActive">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="lblComment">
       <property name="text">
        <string>&amp;Comment:</string>
       </property>
       <property name="buddy">
        <cstring>pteComment</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPlainTextEdit" name="pteComment"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkIndependent">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Independent from the local studio.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btbButtons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ledName</tabstop>
  <tabstop>ledPwd</tabstop>
  <tabstop>ledRePwd</tabstop>
  <tabstop>ledRealName</tabstop>
  <tabstop>chbActive</tabstop>
  <tabstop>pteComment</tabstop>
  <tabstop>btbButtons</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btbButtons</sender>
   <signal>accepted()</signal>
   <receiver>dlgUserEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btbButtons</sender>
   <signal>rejected()</signal>
   <receiver>dlgUserEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
